tests/integration/components/au-modal-test.js

```diff
@@ -202,12 +201,13 @@ module('Integration | Component | au-modal', function (hooks) {
 
     await render(hbs`
       <AuModalContainer />
-      <AuModal @modalOpen={{this.isOpen}} @closeModal={{this.handleClose}}></AuModal>
+      <AuModal @modalOpen={{this.isOpen}} @closeModal={{this.handleClose}} @closable={{false}}></AuModal>
     `);
 
-    let backdrop = document.querySelector(MODAL.BACKDROP);
-    await click(backdrop);
-    assert.strictEqual(timesCalled, 1);
+    assert.dom(MODAL.CLOSE).isDisabled();
```

Comment:
I've had to change this from an actual `click()` to just a `isDisabled()` check because disabled elements are apparently not clickable within tests. Cypress has something along `click({force: true})` but I haven't found anything similar yet.

Reply:
If you can think of a better or more proper solution, please tell!